ui/src/app/store/general/types.ts

```diff
@@ -75,9 +75,9 @@ export type MachineActionName =
 
 export type MachineAction = {
   name: MachineActionName;
-  sentence: string;
-  title: string;
-  type: string;
+  sentence?: string;
```

Comment:
Do these still need to be optional if we're explicitly using `MachineAction["name"]` in the `SelectedAction` type?